corehq/apps/app_manager/models.py

```diff
@@ -2228,7 +2228,7 @@ class CaseSearch(DocumentSchema):
     data_registry = StringProperty(exclude_if_none=True)
     data_registry_workflow = StringProperty(exclude_if_none=True)  # one of REGISTRY_WORKFLOW_*
     additional_registry_cases = StringListProperty()               # list of xpath expressions
-    title_label = DictProperty()
+    title_label = DictProperty(default={'en': 'Case Claim'})
```

Comment:
This should just default to an empty dict, and then add in the default English translation later on, after existing projects are migrated.

Right now, if a live project has updated their English translation of `case.claim.title` in `translations` (which USH projects definitely have), the logic in `_get_label_dict` will override it with this default. There's no way to tell whether a value in `title_label` is there just because it's the default, so you can't set a default translation here until we've moved all of the user-created values from `translations` into `title_label`.

Reply:
would a line like this do the trick?
`label_dict.update(default_label_dict)`

This is assuming `title_lable={}`

Reply:
yes

Reply:
Seeing the code altogether, one more adjustment: right now you're initially setting `label_dict` to the app-level translations, then updating it with a menu's translations, then reusing it for the next menu. That makes it possible for the translations from one menu to get replicated into another when they shouldn't be. If you move the `label_dict` creation inside the for loop, you'll re-grab the app-level translations each time.